src/components/TresCanvas.vue

```diff
@@ -87,6 +88,7 @@ const slots = defineSlots<{
 }>()
 
 const instance = getCurrentInstance()?.appContext.app
+extend(THREE)
```

Comment:
(Non-blocking)

I think this is coupled with `nodeOps`, right? (Like, if we created a totally different Vue renderer, we wouldn't need this at all.)

So, can it be close to line 111 or else inside of `nodeOps`?

Reply:
> If we created a totally different Vue renderer for something besides THREE, we wouldn't need this at all.

Do we? If we would create a custom renderer for, dunno, Babylon, it should exist in a completely different package.